### DISCUSSION TITLE
Net Framework for ArcGis 10.6

### DISCUSSION THREAD

**Discussion by RubenPerezBarcala** [Riverscapes/gcd#440] [I need help please]
Hello, I have a problem with the net famework. Which one to download for ArcGis 10.6. Thanks.

**Comment by joewheaton** [comment]
@RubenPerezBarcala I'm not sure if I understand your question. As long as your on Windows 10 or 11, the .NET framework that ships with should be fine. @philipbaileynar am I mistaken?

**Comment by philipbaileynar** [comment]
What error are you getting?

**Reply by RubenPerezBarcala** [reply]
Hello, thanks for answering.

I have downloaded version 7.5 of your GDC software on Arcgis 10.6, but there is an error with the net framework, despite having windows 10. What could I do? Thank you
